### ISSUE TITLE
docs: update infos on mysql baseline

### ISSUE THREAD

**Issue by hustjieke** [stoneatom/stonedb#1335]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Description

The mysql baselint infos in Docs/INFO_SRC is still 5.6.24, it should be updated.

### Self-service

- [X] I'd be willing to address this documentation request myself.

**Comment by hustjieke** [comment]
```
commit: 97543660999308893335d81b92307c389e6821a5
date: 2015-03-25 17:22:45 +0100
build-date: 2015-03-25 17:26:48 +0100
short: 9754366
branch: mysql-5.6.24-release

MySQL source 5.6.24
```

**Comment by hustjieke** [comment]
Emmm...
This file is meaningless, it should be removed.

**Comment by hustjieke** [comment]
I fixed it in #1336, I'll close this issue now.
